lib/gamera/page_sections/table.rb

```diff
@@ -97,19 +97,18 @@ def initialize(headers:,
                      row_name:,
                      plural_row_name: nil,
                      name_column: 'Name',
-                     row_css: 'tr + tr', # all tr's except the first one (which is almost always a table header)
+                     row_css: 'tr + tr', # all <tr>s except the first one (which is almost always a table header)
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method definition.